### ISSUE TITLE
polyfill.js v0.1.0 with  jQuery v1.11.1 error in IE8

### ISSUE THREAD

**Issue by hyunjun19** [inexorabletash/polyfill#51]
An error occurs when the page is loaded.

`SCRIPT5011: Can't execute code from a freed script`

![polyfill-error-01](https://cloud.githubusercontent.com/assets/421863/5950387/d86e72bc-a79c-11e4-98ca-876ecf388944.png)
![polyfill-error-02](https://cloud.githubusercontent.com/assets/421863/5950388/da257312-a79c-11e4-8641-78c45f18ad46.png)
![polyfill-error-03](https://cloud.githubusercontent.com/assets/421863/5950390/dbf1e4d2-a79c-11e4-8c6f-c15c85b6ca82.png)

**Comment by inexorabletash** [comment]
Any chance you can provide a repro?

The following isn't enough to demonstrate the problem (with IE11 in IE8 mode):

```html
<!DOCTYPE html>
<script src="https://inexorabletash.github.io/polyfill/polyfill.js"></script>
<script src="https://code.jquery.com/jquery-1.11.1.js"></script>
```

Catching/ignoring that error in removeEvent is harmless, but I'd prefer to be able to reproduce it to make sure.

**Comment by inexorabletash** [comment]
Huh... I thought I remembered fixing this, actually. There's a commit with no changeset but the comment "IE8-mode fix for IE11 debugger" c6be767cc032e9344dc1ad63c9337327264a7cf0 - I wonder what happened to it...

**Comment by hyunjun19** [comment]
There is no problem in IE8.
IE11 seems to be a problem that occurs only in emulation mode.
I just decided not to test in IE11 emulation mode.

Thank you very much.
